runtime/jcl/common/system.c

```diff
@@ -140,10 +138,14 @@ Java_java_lang_System_getSysPropBeforePropertiesInitialized(JNIEnv *env, jclass
 	case 2: /* file.encoding */
 		sysPropValue = getDefinedEncoding(env, "-Dfile.encoding=");
 		if (NULL == sysPropValue) {
-#if JAVA_SPEC_VERSION >= 18
-			sysPropValue = "UTF-8";
-#else /* JAVA_SPEC_VERSION >= 18 */
+#if JAVA_SPEC_VERSION < 18
 			sysPropValue = getPlatformFileEncoding(env, property, sizeof(property), sysPropID);
+#else /* JAVA_SPEC_VERSION >= 18 */
+			sysPropValue = "UTF-8";
```

Comment:
Why `UTF-8` here, but `UTF8` (no dash) on line 166?

Reply:
Either works but `UTF-8` is the standard name. `UTF8` is the historical name that gets returned in some cases, and an alias.

Reply:
I'm just suggesting we be consistent. If either is the canonical name, we should use that throughout.

Reply:
Changed existing L166 to `UTF-8`.

Reply:
We should also use `"ISO-8859-1"` rather than the alias `"ISO8859_1"`.